src/pages/components/table.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: Table
+description: >
+  A Table organizes data into columns and rows. Tables make it easier for users
+  to compare and analyze information.
+package: '@zendeskgarden/react-tables'
+components:
+  - tables/src/elements/Body.tsx
+  - tables/src/elements/Caption.tsx
+  - tables/src/elements/Cell.tsx
+  - tables/src/elements/GroupRow.tsx
+  - tables/src/elements/Head.tsx
+  - tables/src/elements/HeaderCell.tsx
+  - tables/src/elements/HeaderRow.tsx
+  - tables/src/elements/OverflowButton.tsx
+  - tables/src/elements/Row.tsx
+  - tables/src/elements/SortableCell.tsx
+  - tables/src/elements/Table.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To organize and display data with logical relationships</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To structure content on a page, use the [Grid](/components/grid) component instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Table component.
+
+import Default from '../../examples/table/TableDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/table/TableDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Grouped rows
+
+Grouped rows are for additional categorization.
+
+import GroupedRows from '../../examples/table/TableGroupedRows.tsx';
+import GroupedRowsCode from '!!raw-loader!../../examples/table/TableGroupedRows.tsx';
+
+<CodeExample code={GroupedRowsCode}>
+  <GroupedRows />
+</CodeExample>
+
+### Overflow
+
+Overflow menus leverage the Dropdown and Menu component.
+
+import Overflow from '../../examples/table/TableOverflow.tsx';
+import OverflowCode from '!!raw-loader!../../examples/table/TableOverflow.tsx';
+
+<CodeExample code={OverflowCode}>
+  <Overflow />
+</CodeExample>
+
+### Pagination
+
+Table pagination works in combination with the [Pagination](/components/pagination) component.
+
+import Pagination from '../../examples/table/TablePagination.tsx';
+import PaginationCode from '!!raw-loader!../../examples/table/TablePagination.tsx';
+
+<CodeExample code={PaginationCode}>
+  <Pagination />
+</CodeExample>
+
+### Scroll
+
+To ensure that your table is read correctly by assistive technologies follow the
+[W3C Grid accessibility
+pattern](https://www.w3.org/TR/wai-aria-practices-1.1/examples/grid/dataGrids.html).
+
+import Scroll from '../../examples/table/TableScroll.tsx';
+import ScrollCode from '!!raw-loader!../../examples/table/TableScroll.tsx';
+
+<CodeExample code={ScrollCode}>
+  <Scroll />
+</CodeExample>
+
+### Selection
+
+Individual rows within a table can be selected for bulk actions.
+
+import SelectRows from '../../examples/table/TableSelectRows.tsx';
+import SelectRowsCode from '!!raw-loader!../../examples/table/TableSelectRows.tsx';
```

Comment:
Also realized that this example allows for multiple row selection. On the original set of docs, we say that this is additional functionality… should we do the same here?